benchmarks/bench.sh

```diff
@@ -318,6 +338,62 @@ run_sort() {
     $CARGO_COMMAND --bin parquet -- sort --path "${DATA_DIR}" --scale-factor 1.0 --iterations 10 -o ${RESULTS_FILE}
 }
 
+
+# Downloads the single file hits.parquet ClickBench datasets from
+# https://github.com/ClickHouse/ClickBench/tree/main#data-loading
+#
+# Creates data in $DATA_DIR/hits.parquet
+data_clickbench_1() {
+    pushd "${DATA_DIR}" > /dev/null
+
+    # Avoid downloading if it already exists and is the right size
+    OUTPUT_SIZE=`wc -c hits.parquet  2>/dev/null  | awk '{print $1}' || true`
+    echo -n "Checking hits.parquet..."
+    if test "${OUTPUT_SIZE}" = "14779976446"; then
+        echo -n "... found ${OUTPUT_SIZE} bytes ..."
+    else
+        URL="https://datasets.clickhouse.com/hits_compatible/hits.parquet"
+        echo -n "... downloading ${URL} (14GB) ... "
```

Comment:
Yes, you are right, they should both by bytes (KB was left over from a previous implementation). Nice eyes

Reply:
> Do you mean the other way? Since OUTPUT_SIZE is constant in the if statement and dynamic in the else statement

I think this is the right way -- basically this code is testing if a file with the expected size already exists, and if so it doesn't download it again.

Reply:
> This means that you can run `bench.sh download clickbench_1` and be sure you have the most recent data but not have to download all 14GB if it already exists

Ah, got it! Thank you for the explanation